test/views/VatReturnDetailsViewSpec.scala

```diff
@@ -558,18 +565,54 @@ class VatReturnDetailsViewSpec extends ViewBaseSpec {
       showReturnsBreadcrumb = true,
       currentYear,
       hasFlatRateScheme = true,
+      isOptOutMtdVatUser = false,
       isHybridUser = false
     )
 
-    lazy val view = views.html.returns.vatReturnDetails(vatReturnViewModel)
-    lazy implicit val document: Document = Jsoup.parse(view.body)
 
-    "have the pageHeading of 'Final return'" in {
-      elementText(Selectors.pageHeading) shouldBe "Submitted returns Final return"
-    }
+    "Rendering the VAT return details page when the user is opted out" should {
```

Comment:
This fails as you've accidentally put this test inside the one above - please move into its own :)